src/search/interfaces.ts

```diff
@@ -0,0 +1,12 @@
+export type Metadata = Array<MetaItem>;
+
+export type MetaItem = {
```

Comment:
```suggestion
export type MetaItem = {
  [index: string]: string | string[];
```

Reply:
This should let you access this in the normal way?

Reply:
This is the fix I was looking for! Thanks!